src/GovUk.Education.ExploreEducationStatistics.Publisher/Startup.cs

```diff
@@ -52,7 +56,17 @@ public override void Configure(IFunctionsHostBuilder builder)
                     options.UseSqlServer(ConnectionUtils.GetAzureSqlConnectionString("PublicStatisticsDb")))
                 .AddSingleton<IFileStorageService, FileStorageService>(provider =>
                     new FileStorageService(GetConfigurationValue(provider, "PublisherStorage")))
-                .AddScoped(provider => GetBlobCacheService(provider, "PublicStorage"))
+                .AddScoped(provider =>
+                {
+                    var publicBlobCacheService = GetBlobCacheService(provider, "PublicStorage");
+                    CacheAspect.Enabled = true;
+                    BlobCacheAttribute.AddService("default", publicBlobCacheService);
+                    return publicBlobCacheService;
+                })
+                .AddScoped<IContentMethodologyService, ContentMethodologyService>()
+                .AddScoped<IMethodologyCacheService, MethodologyCacheService>()
+                .AddScoped<IThemeCacheService, ThemeCacheService>()
+                .AddScoped<IThemeService, ThemeService>()
```

Comment:
`ThemeService` hasn't been aliased `ContentThemeService` like it is for `ContentMethodologyService`, and the same for the same services in the Admin's `Startup.cs`. I see there's no `ThemeService` in Publisher for it to clash with, but I think it would be good name these consistently.